src/Command/DkanCommands.php

```diff
@@ -169,10 +169,35 @@ public function dkanDemo()
             ->exec("dktl frontend:build")
             ->exec("dktl drush cr")
             ->run();
-        
+
         $this->io()->success("Your demo site is available at: " . Util::getUri());
     }
 
+    /**
+     * Create a new dev project.
+     *
+     * Will have frontend and sample content. Run this immediately after dktl
+     * init.
+     *
+     * @aliases dev
+     */
+    public function dkanDev()
+    {
+        $this->taskExecStack()
+            ->stopOnFail()
+            ->exec("dktl make --prefer-source")
+            ->exec("dktl install")
+            ->exec("dktl install:sample")
+            ->exec("git clone https://github.com/getdkan/data-catalog-app src/frontend")
```

Comment:
Looking at the frontend class, seems we do have variables with the wanted frontend repo and branch. Guessing it would be better to re-use those.
```suggestion
            ->exec("git clone " . " -b " . FrontendCommands::FRONTEND_VCS_REF . " " . FrontendCommands::FRONTEND_VCS_URL . " " . "src/frontend")
```